resources/splunk_app.rb

```diff
@@ -69,25 +65,43 @@ def parse_meta_access(access)
     end
   end
 
+  # Provider exclusive methods
+
   action_class do
     include CernerSplunk::ProviderHelpers
 
+    def upgrade_app(strategy)
+      require 'fileutils'
```

Comment:
It probably makes no difference if this is required outside. This method is not called in every instance of the resource, however.